OWML.ModLoader/Owo.cs

```diff
@@ -38,8 +42,28 @@ public void LoadMods()
                 Application.logMessageReceived += OnLogMessageReceived;
             }
             var mods = _modFinder.GetMods();
-            foreach (var modData in mods)
+
+            var sortedMods = ModSorter.SortMods(mods);
```

Comment:
I would prefer ModSorter to not be static. It can be newed up outside of Owo and injected in the constructor.